apps/front-end/src/components/map/markers.ts

```diff
@@ -0,0 +1,16 @@
+import defaultMarker from "./markers/map-marker.png";
+import communityGrowingMarker from "./markers/community-growing-marker.png";
+import emergencyFoodMarker from "./markers/emergency-food-marker.png";
+import eventsMarker from "./markers/events-marker.png";
+import localFoodMarker from "./markers/local-food-marker.png";
+
+//when adding new markers, ensure default is still the last entry
+const markers = {
```

Comment:
Might be worth leaving a big notice in a comment that these labels are hardwired and this is just a temporary situation, to be rectified later.